### PR TITLE
Task 26

### PR DESCRIPTION
**Pull request by joergenreyes**
Pull request to merge 'joergen' into 'main'

### CODE REVIEW

File: -sass/styles/_chip.scss [hunk 1]
```diff
@@ -0,0 +1,48 @@
+.s-chip {
+    --chip-height: 30px;
+    
+    background-color: $light-grey;
```
Comment (pmvald) [inline -sass/styles/_chip.scss:4]: maybe define this by a css variable like `var(--bg-color, $light-grey)`

File: -sass/styles/_chip.scss [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    display: inline-flex;
+    align-items: center;
+    border-radius: 1000px;
```
Comment (pmvald) [inline -sass/styles/_chip.scss:7]: maybe `border-radius: var(--chip-height)`?

**Review by pmvald — CHANGES_REQUESTED** [review]
@joergenreyes I made some code review - please attend - thanks

**Comment by joergenreyes** [comment]
@pmvald I have implemented the changes to the SCSS chips document.

**Review by pmvald — APPROVED** [review]

**Comment by pmvald** [comment]
@joergenreyes cool! thanks!

**Pull request merged**